### ISSUE TITLE
Stationary distribution of hidden states and emissions

### ISSUE THREAD

**Issue by anntzer** [hmmlearn/hmmlearn#96]
An API for retrieving the stationary distribution of hidden states and emissions could be useful.  Something like:

```
hmm.get_stationary_distribution() -> (n_states,) array
hmm.get_emission_pdf(x) -> (n_states, len(x)) array
```

Thoughts?

**Comment by superbobry** [comment]
Could you elaborate a bit on what you expect these to return?

**Comment by anntzer** [comment]
`get_stationary_distribution` should basically compute the left eigenvector of the transition matrix associated with the eigenvalue 1 (https://en.wikipedia.org/wiki/Markov_chain#Steady-state_analysis_and_limiting_distributions), mostly, well, to know what the stationary distribution is (which is typically one of the first things I check after fitting an HMM).

`get_emission_pdf(x)` where `x` is a `(n, n_features)` array (or the other way round depending on conventions) should compute, for each entry (emission) of `x` and for each state, the pdf in the stationary distribution of emissions to observe such an emission, multiplied by the probability that such an emission comes from a given state.  The goal is to create plots such as figures 2 and 3 of http://arxiv.org/pdf/1108.1430v1.pdf which are fairly common at least in the single-molecule biophysics community.

**Comment by anntzer** [comment]
Closed by #141.
